src/test/java/com/hotel/api/book/service/HotelBookServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package com.hotel.api.book.service;
+
+import com.hotel.api.book.mappers.book.BookRequestBuilder;
+import com.hotel.api.book.model.book.request.BookRequest;
+import com.hotel.api.book.model.book.request.PaymentInfo;
+import com.hotel.service.book.HotelBookRequest;
+import com.hotel.service.book.HotelBookResponse;
+import com.hotel.service.book.HotelBookServiceGrpc;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class HotelBookServiceTest {
+
+    @Mock
+    private HotelBookServiceGrpc.HotelBookServiceBlockingStub hotelBookServiceBlockingStub;
+
+    @Mock
+    private BookRequestBuilder bookRequestBuilder;
+
+    @InjectMocks
+    private HotelBookService hotelBookService;
+
+    @Test
+    public void service() throws Exception {
+        BookRequest request = new BookRequest();
```

Comment:
Already Book Request object created in BookRequestBuilderTest please reuse it here